online-shop/src/HelperComponents/ConfirmationModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import '../Styles/ComponentsStyles/ConfirmationModal.scss';
+import { Redirect } from 'react-router';
+
+interface IProps {
+    match?: any;
+    ProductId: number;
+    show: boolean;
+    showModalFunction: any;
+}
+
+interface IState {
+    shouldRedirect: boolean;
+}
+
+export default class ConfirmationModal extends React.Component<IProps, IState> {
+    abortController = new AbortController();
+
+    constructor(props: IProps) {
+        super(props);
+
+        this.state = {
+            shouldRedirect: false
+        }
+    }
+
+    handleDeleteProductClick = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
I think you don't need this parameter -> you can just omit it from the function definition.